### PR TITLE
runtime: Fix config.md -> config.json typo

### PR DESCRIPTION
**Pull request by wking**
Typo from #559.

### CODE REVIEW

**Comment by mrunalp** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/opencontainers/runtime-spec/pull-request/643/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=opencontainers/runtime-spec)

File: runtime.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ This operation MUST generate an error if it is not provided a path to the bundle
 If the ID provided is not unique across all containers within the scope of the runtime, or is not valid in any other way, the implementation MUST generate an error and a new container MUST NOT be created.
 Using the data in [`config.json`](config.md), this operation MUST create a new container.
 This means that all of the resources associated with the container MUST be created, however, the user-specified program MUST NOT be run at this time.
-If the runtime cannot create the container as specified in [`config.md`](config.md), it MUST generate an error and a new container MUST NOT be created.
+If the runtime cannot create the container as specified in [`config.json`](config.md), it MUST generate an error and a new container MUST NOT be created.
```
Comment (crosbymichael) [inline runtime.md]: What about the `(config.md)` in the same line?

Comment (wking) [inline runtime.md]: That's a Markdown link target, so folks can see the spec behind `config.json`.

**Comment by dqminh** [comment]
LGTM

**Comment by hqhq** [comment]
LGTM

**Pull request merged**